src/aws.ts

```diff
@@ -52,31 +42,31 @@ export class AWS {
     }
 
     public async getExistingFiles(bucket: string, prefix?: string) {
-        async function existingFilesKeys(
-            s3: S3,
-            param: ListObjectsV2Request,
-            allKeys: string[] = [],
-        ): Promise<string[]> {
+        async function existingFilesKeys(s3: S3, param: ListObjectsV2Request, allKeys: string[] = []): Promise<string[]> {
             return new Promise((resolve, reject) => {
                 s3.listObjectsV2(param, async (error: AWSError | null, data) => {
                     if (error) {
                         reject(error)
                     } else {
                         if (data.Contents) {
-                            allKeys.push(...data.Contents.map((content) => {
-                                return content.Key as string
-                            }))
+                            allKeys.push(
+                                ...data.Contents.map((content) => {
+                                    return content.Key as string
```

Comment:
It would be better to check if `content.Key` exists and throw an error if it doesn't instead of casting.

Reply:
i didn't change it, it's my automatic linter sorry

Reply:
Fixed